packages/fuels-abigen-macro/tests/harness.rs

```diff
@@ -1221,6 +1229,8 @@ async fn test_multiple_args() {
 
 #[tokio::test]
 async fn test_tuples() {
+    setup_logger();
```

Comment:
why is it used only here?

Reply:
https://github.com/FuelLabs/fuels-rs/issues/244 
This issue requires that the results of the receipts be returned. I have used this here just to show how it is used and what results it gives. When I'm done I'll document it in the FuelRs book.

Reply:
if this setup is needed throughout the test suite, we should initialize it once for all the tests, maybe [ctor ](https://docs.rs/ctor/latest/ctor/)crate could help with it.